README.md

```diff
@@ -0,0 +1,154 @@
+
+```npx fluid-lint-all```
+
+Although it's not required, it is recommended that you install the
+[standard ESLint configuration used within the community](https://github.com/fluid-project/eslint-config-fluid).  If you
+choose not to, you must at least have some kind of `.eslintrc.json` in your repository.  Otherwise, some of the ESLint
+checks will fail for lack of a configuration file.
+
+## Tests
+
+To run the tests in this package, use a command like `npm test`.
+
+## Checks Available
+
+| Task                       | Description |
+| -------------------------- | ----------- |
+| eslint                     | Use [ESLint](https://eslint.org) to lint both javascript and markdown content. |
+| eslint.js                  | Check the validity and formatting of Javascript files. |
+| eslint.md                  | Check the validity and formatting of Javascript code blocks in Markdown files. |
+| json5lint                  | Check the validity of JSON5 files. |
+| jsonlint                   | Use [jsonlint](https://www.npmjs.com/package/jsonlint) to check the validity and formatting of JSON files. |
+| lintspaces                 | Use [lintspaces](https://github.com/schorfES/node-lintspaces) to check for trailing carriage returns and JSON indentation issues. |
+| lintspaces.jsonindentation | Check the indentation of JSON files. |
+| lintspaces.newlines        | Check for the presence of a carriage return at the end of a file. |
+| markdownlint               | Use [markdownlint](https://github.com/DavidAnson/markdownlint) to check the formatting of Markdown files. |
+| mdjsonlint                 | Check the validity and formatting of JSON code blocks within Markdown files. |
+| stylelint                  | Use [stylelint](https://stylelint.io) to check the formatting of CSS or SCSS files.|
+
+## Configuration
+
+This package is configured using a JSON file, which by default is named
+`.fluidlintallrc.json` and located in the root of your repository.  You can run the checks using
+a different configuration file by passing the `configFile` option, as in
+`npx fluid-lint-all --configFile ./path/to/your/config.json`.
+
+You can choose to disable any checks that are not relevant to your environment
+in the configuration file (see below), but you can also specify which check(s)
+to run by passing the `checks` option, as in `npx lint-all --checks=eslint` or
+`npx fluid-lint-all --checks=jsonlint,lintspaces.newlines`
+
+Each check supports the following configuration options.
+
+| Option   | type            | Description |
+| -------- | --------------- | ----------- |
+| enabled  | `Boolean`       | Whether or not to run the check.  If set to `false`, will also disable any sub-checks.  Defaults to `true`. |
+| includes | `Array<string>` | An array of glob patterns indicating files to be linted. |
+| excludes | `Array<string>` | An array of glob patterns indicating files (that otherwise match the includes) which should be excluded. |
```

Comment:
As per @BlueSlug (https://github.com/fluid-project/fluidic-11ty/pull/75#pullrequestreview-569380254), `fluid-lint-all` appears to be linting a JSON file which is present in the project's `.gitignore`. I'm wondering what your thoughts are on this, @the-t-in-rtf? My expectation would be that files which are not committed to version control should be excluded by default, but open to other opinions.

Reply:
I can see why you'd want something like this, as the assumption is that you only need to lint things you're checking in and not local generated cruft.  I'd like to get the initial release out the door first, if you could write it up as a separate small issue I'm sure I can find time for a follow-up minor release.  My thinking is that we'd have a flag that's set by default to automatically add .gitignore material to all excludes.